templates/default/voipaccount/voipaccountbillinglist.html

```diff
@@ -189,6 +189,8 @@ <h1>{$layout.pagetitle}</h1>
 							{if $listdata.ftype == $smarty.const.BILLING_RECORD_TYPE_MMS} selected{/if}>{trans("<!voip>MMS")}</option>
 						<option value="{$smarty.const.BILLING_RECORD_TYPE_DATA_TRANSFER}"
 							{if $listdata.ftype == $smarty.const.BILLING_RECORD_TYPE_DATA_TRANSFER} selected{/if}>{trans("<!voip>data transfer")}</option>
+						<option value="{$smarty.const.BILLING_RECORD_TYPE_VIDEO}"
```

Comment:
W takich miejscach jako to używajmy nowych nazw stałych. Tylko dodatki nam nieznane mogą korzystać ze starych nazw stałych. To w odniesieniu do BILLING_RECORD_TYPE_VOICE_CALL i BILLING_RECORD_TYPE_CALL, a w przypadku BILLING_RECORD_TYPE_VIDEO to stałej nawet nie powinno być, bo nigdy jej wcześniej nie było i nie ma zagrożenia zerwania kompatybilności.